src/vm/comtoclrcall.cpp

```diff
@@ -782,7 +781,7 @@ extern "C" UINT64 __stdcall COMToCLRWorker(Thread *pThread, ComMethodFrame* pFra
 
 #ifndef _TARGET_X86_
 ErrorExit:
-    if (pThread->PreemptiveGCDisabled())
+    if (pThread != nullptr && pThread->PreemptiveGCDisabled())
```

Comment:
Nit: It would be nice to either use NULL in both places or nullptr in both places.